Products/CMFPlone/tests/testEmailLogin.py

```diff
@@ -83,8 +83,9 @@ def testEmailMemberIdsAllowed(self):
         # email addresses.
         # A plus sign in the id gives problems in some parts of the
         # UI, so we do not allow it.
```

Comment:
Can you remove this comment?  It is outdated after your changes.

Reply:
done